src/i18n/ES/grid-resources.ts

```diff
@@ -0,0 +1,76 @@
+import { IGridResourceStrings } from 'igniteui-angular';
+
+/**
+ * Spanish resource strings for IgxGrid
+ */
+export const GridResourceStringsES: IGridResourceStrings = {
+  igx_grid_groupByArea_message: 'Arrastre un encabezado de columna y suéltelo aquí para agrupar por esa columna.',
+  igx_grid_emptyFilteredGrid_message: 'No se encontraron Registros.',
+  igx_grid_emptyGrid_message: 'El Grid no tiene Datos.',
+  igx_grid_filter: 'Filtro',
+  igx_grid_filter_row_close: 'Cerrar',
+  igx_grid_filter_row_reset: 'Reiniciar',
+  igx_grid_filter_row_placeholder: 'Añadir un Valor para el Filtro',
+  igx_grid_filter_row_boolean_placeholder: 'Todos',
+  igx_grid_filter_row_date_placeholder: 'Fecha de Recogida',
+  igx_grid_filter_operator_and: 'Y',
+  igx_grid_filter_operator_or: 'O',
+  igx_grid_complex_filter: 'Filtro Complejo',
+  igx_grid_filter_contains: 'Contiene',
+  igx_grid_filter_doesNotContain: 'No Contiene',
+  igx_grid_filter_startsWith: 'Empieza Con',
+  igx_grid_filter_endsWith: 'Termina Con',
+  igx_grid_filter_equals: 'es Igual',
+  igx_grid_filter_doesNotEqual: 'No es Igual',
+  igx_grid_filter_empty: 'Vacío',
+  igx_grid_filter_notEmpty: 'No Vacío',
+  igx_grid_filter_null: 'Nulo',
+  igx_grid_filter_notNull: 'No Nulo',
+  igx_grid_filter_before: 'Antes',
+  igx_grid_filter_after: 'Despues',
+  igx_grid_filter_today: 'Hoy',
+  igx_grid_filter_yesterday: 'Ayer',
+  igx_grid_filter_thisMonth: 'Este Mes',
+  igx_grid_filter_lastMonth: 'El Mes Pasado',
+  igx_grid_filter_nextMonth: 'El Próximo Mes',
+  igx_grid_filter_thisYear: 'Este Año',
+  igx_grid_filter_lastYear: 'El Año Pasado',
+  igx_grid_filter_nextYear: 'El Próximo Año',
+  igx_grid_filter_greaterThan: 'Mayor que',
+  igx_grid_filter_lessThan: 'Menor que',
+  igx_grid_filter_greaterThanOrEqualTo: 'Mayor o Igual que',
+  igx_grid_filter_lessThanOrEqualTo: 'Menor o igual que',
+  igx_grid_filter_true: 'Verdadero',
+  igx_grid_filter_false: 'Falso',
+  igx_grid_filter_all: 'Todas',
+  igx_grid_filter_condition_placeholder: 'Seleccionar Filtro',
+  igx_grid_summary_count: 'Cuenta',
+  igx_grid_summary_min: 'Min',
+  igx_grid_summary_max: 'Max',
+  igx_grid_summary_sum: 'Sum',
+  igx_grid_summary_average: 'Promed',
+  igx_grid_summary_earliest: 'El mas temprano',
+  igx_grid_summary_latest: 'El último',
+  igx_grid_excel_filter_moving_left: 'izquierda',
+  igx_grid_excel_filter_moving_right: 'derecha',
+  igx_grid_excel_filter_moving_header: 'moviendo columna',
+  igx_grid_excel_filter_sorting_asc: 'ascendiendo',
+  igx_grid_excel_filter_sorting_desc: 'descendiendo',
+  igx_grid_excel_filter_sorting_header: 'clasificando',
+  igx_grid_excel_custom_dialog_add: 'añadir filtro',
+  igx_grid_excel_custom_dialog_clear: 'Limpiar Filtro',
+  igx_grid_excel_custom_dialog_header: 'Auto-filtro personalizado en una columna: ',
```

Comment:
I think this text might be too long.
@bkulov is this modal width fixed?
![image](https://user-images.githubusercontent.com/12942794/57093149-7fdbd480-6ce3-11e9-81f3-b18b3a3b5b83.png)

Maybe we can put just "Auto-filtro personalizado para"
i.e. Omit the "column" word.

Reply:
I've tested how this string looks like. If the column name is long it would be wrapped on a new line (only the column name). So if this label could be shortened it would be good. But if the short string doen't make sense we could still use the long one.

Reply:
I think it does make sense, without the Column word.
"Auto-filtro personalizado para: "